src/pykx/pandas_api/pandas_meta.py

```diff
@@ -134,6 +134,11 @@ def shape(self):
     @property
     def size(self):
         return q('{count[x] * count[cols x]}', self)
+    
```

Comment:
Running `pflake8` returns "W293 blank line contains whitespace" on this line.

Reply:
Working on it, thanks